src/functionalTest/resources/application-functional.yaml

```diff
@@ -9,4 +9,4 @@ oauth2.client.secret: ${CA_REF_OAUTH2_CLIENT_SECRET:}
 idam.auth.clientId: ${CA_REF_OAUTH2_CLIENT_ID:}
 idam.auth.redirectUrl: ${OAUTH2_REDIRECT_URI:https://rd-caseworker-ref-api-aat.service.core-compute-aat.internal/oauth2redirect}
 s2s-secret: ${S2S_SECRET:}
-
+userProfUrl: https://rd-user-profile-api-pr-575.service.core-compute-preview.internal
```

Comment:
Should not we use aat url here instead of preview?

Reply:
Currently, the UP has not whitelisted the CW Ref api. Hence used the PR 575. Before going into higher env, we should change it to aat.

Reply:
Should we track this in a ticket then, just in case if we missed to update ? FYI - @pksaxena01

Reply:
Good thought. Let's track it in RDCC-2059 ticket. Thanks 👍

Reply:
Thanks for adding in the ticket